PlaceholdersJumping.md

```diff
@@ -1 +1,142 @@
-Write article
\ No newline at end of file
+# Paging 3's Placeholders and Jumping Features
+
+(Royalty-free hipstery photo or art showcasing pages)
+
+With the release of Paging 3, a host of new features have been made available: from first-class support for coroutines and Flow to data stream transformations, to improvements for reactive loading state. Most of these features, alongside migration guides, are prominently featured on the official documentation. However, some others and returning ones are just briefly mentioned or omitted from the CodeLabs for the sake of simplicity. In this article, I'm hoping to expand a bit on two of these: placeholders and the jumping functionality they enable.
+
+Note: This article assumes the reader has basic knowledge of the Paging 3 library. Checking the official documentation (https://developer.android.com/topic/libraries/architecture/paging/v3-overview) is a great way to get started.
+
+
+## Data set requirements
+
+The most important point to consider whenever attempting to implement placeholders and jumping with Paging 3 is that our data set needs to be countable, this means that we should know exactly how many items the list can have in its entirety. If your data set source is completely network-based, your API responses should provide you with a count property or another mechanism to get this metadata.
+
+```
+{
+  "total": 3391, // This is what we want
+  "items": [
+    ...
+  ]
+}
+```
+
+## Placeholders
+
+Placeholders are UI elements that we can show while awaiting the loading of our real pages and items. They make for a nice UI pattern where lists are uninterrupted and user interaction is not limited by loading new elements. Aside from that, because we can show the entire list from the get-go, we are also able to better support scrollbars and, after the first page is loaded, there's no longer the need to show loading spinners on footers. Instead, the placeholders themselves convey this scenario.
```

Comment:
Essentially a loading footer == placeholder that is always placed in the end of the list, whereas with non linear loading the 'footer' can be anywhere.